src/mumble/ConnectDialog.cpp

```diff
@@ -473,11 +473,23 @@ QVariant ServerItem::data(int column, int role) const {
 					return uiUsers ? QString::fromLatin1("%1/%2 ").arg(uiUsers).arg(uiMaxUsers) : QVariant();
 			}
 		} else if (role == Qt::ToolTipRole) {
-			QStringList qsl;
+			QStringList qslIpv4;
+			QStringList qslIpv6;
```

Comment:
Please don't overtake the variable naming scheme that is found in many places in Mumble where a variable's type is encoded in some prefixing mess in the variable name. This is generally bad practice and we are currently trying to get rid of that.

Just call these e.g. `ipv4Addresses` or something like that.